Job warnings (#5690)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

## June 2024

> - **New:** GA of job warnings is now available. Previously, you could receive email or Slack alerts about your jobs when they succeeded, failed, or were cancelled. Now, you can also receive alerts when jobs have encountered warnings during their run. This provides more flexibility on when to be notified. 

Author: @reubenmc : not sure about the wording here. would love your 👀 on it

> - **New:** GA of [job warnings](/docs/deploy/job-notifications) is now available. Previously, you could receive email or Slack alerts about your jobs when they succeeded, failed, or were cancelled. Now, you can also receive alerts when jobs have encountered warnings during their run. This provides more flexibility on when to be notified. 

Reviewer: Love this! Can we change it to something like "when jobs have warnings from tests or source freshness checks"

> - **New:** GA of [job warnings](/docs/deploy/job-notifications) is now available. Previously, you could receive email or Slack alerts about your jobs when they succeeded, failed, or were canceled. Now with the new **Warns** option, you can also receive alerts when jobs have encountered warnings from tests or source freshness checks during their run. This gives you more flexibility on _when_ to be notified. 

Reviewer 2: ```suggestion
- **New:** [Job warnings](/docs/deploy/job-notifications) are now GA. Previously, you could receive email or Slack alerts about your jobs when they succeeded, failed, or were canceled. Now with the new **Warns** option, you can also receive alerts when jobs have encountered warnings from tests or source freshness checks during their run. This gives you more flexibility on _when_ to be notified. 
```

Reviewer: Love it!
